app/views/layouts/_main_nav.html.erb

```diff
@@ -1,7 +1,7 @@
 <% nav_links = NavigationLink.all.ordered.to_a %>
 
-<nav class="mb-6 <% unless user_signed_in? %>mt-4<% end %>" id="sidebar-nav" aria-label="Primary sidebar nav">
-  <a href="<%= root_path %>" class="crayons-link crayons-link--block">
+<nav class="mb-6 <% unless user_signed_in? %>mt-4<% end %>" aria-label="Primary sidebar nav">
+  <a href="<%= root_path %>" class="crayons-link crayons-link--block sidebar-nav">
```

Comment:
```suggestion
  <a href="<%= root_path %>" class="crayons-link crayons-link--block">
```

Refactor (blocking) - I don't think this class exists? I don't think the id is used for anything (from what I can tell), so seems like it is safe to remove